### DISCUSSION TITLE
Proposal: Drop provecommitsectors message after ProveCommitSectors2 inclusion

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/FIPs#899] [General]
Initial FIP PR: https://github.com/filecoin-project/FIPs/pull/820

#689 was the original problem that inspired this proposal. Tl'dr is that the per sector activation cost is much more expensive if SPs prove commit the sectors with non-aggregated proof (using `ProveCommitSector` method) comparing to using the aggregated proof (using `ProveCommitAggregate` method). Reason being the activation work using the former method is done asynchronously via cron jobs, and the gas is improperly subsidized by the chain.

To resolved the imbalance in cost of onboarding sectors and remove the inefficient incentive use of chain validation resources caused by discouraging the use of proof aggregation, a solution is needed to move the undesired asynchronous activation job to be  done by the user synchronously.

With the recent FIP-0076's acceptance, the solution becomes trivial: we could simply drop the `ProveCommitSectors` (method 7 in storage miner actor). Thanks to FIP-0076, it introduces [ProveCommitSector3](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0076.md#provecommitsectors3) method that allows synchronous sector activation for non-aggregated proof.  The method include other richer functions as detailed in the FIP-0076, including supporting both batched and aggregated proof and etc,  which makes the old method redundant. To drop the old method completely will also reduced the complexity, overhead and risk in actor/implementations & SP operations.

While the new PCS3 method doesn't support sectors with f05 deals, such sectors could still be onboarded with `ProveCommitAggregate` with discount proof validations on chain or `ProveUpdateReplica` on top of CC sectors. Thus, all functions in Filecoin still remain supported without `ProveCommitSector` method, as long as the removal happens after PCS3's finalization in a network upgrade.

**Comment by Stebalien** [comment]
I'd suggest that we pause last call on this FIP until FIP0076 is live on the network and has been tested at scale.

**Reply by jennijuju** [reply]
thanks I have created https://github.com/filecoin-project/FIPs/pull/927 to capture this. I don't think it has to block the fip governance process as long as we fully understand what t he conditions are
